### ISSUE TITLE
[Insight] Dependencies not installable

### ISSUE THREAD

**Issue by AmiAhm** [vektorprogrammet/vektorprogrammet#1180]
```
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for hhvm 2.4.0-dev -> satisfiable by hhvm[2.4.0-dev].
    - friendsofphp/php-cs-fixer v2.14.0 conflicts with hhvm[2.4.0-dev].
    - Installation request for friendsofphp/php-cs-fixer v2.14.0 -> satisfiable by friendsofphp/php-cs-fixer[v2.14.0].


```

---------------------

_Posted from [SymfonyInsight](https://insight.symfony.com/projects/ec066ae0-5658-4365-af0c-4ab7492d2ad7/analyses/2530)_

**Comment by AmiAhm** [comment]
Brukes disse pakkene i det hele tatt?

**Comment by kristandre** [comment]
Fixed in #1172
